partners/project-19.md

```diff
@@ -0,0 +1,55 @@
+---
+altLangPage: "https://conception.canada.ca/partenaires/project-19.html"
+date: 2024-01-03
+dateModified: 2024-01-03
+description: "Partner project: Make it easier for Canadians to understand notifications about dangerous weather conditions. Date: July 2017."
+title: "Weather information"
+---
+<p>Make it easier for Canadians to understand notifications about dangerous weather conditions.</p>
+<h2>Date</h2>
+<p>July 2017</p>
+<h2>Partner department</h2>
+<ul>
+  <li>Environment and Climate Change Canada</li>
+</ul>
+<h2>Why this work matters</h2>
+<ul>
+  <li>Flu cases result in nearly 12,000 hospitalizations in Canada each year</li>
```

Comment:
This is incorrect. It should be: 

- Canadians check the weather with the Government of Canada more than they do anything else

- More than 550,000,000 visits to weather.gc.ca annually

Reply:
Fixed.